src/test/scala/org/camunda/feel/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -227,10 +227,4 @@ class BuiltinNumberFunctionsTest
     eval(" odd(-2)") should be(ValBoolean(false))
   }
 
-  "A even() function" should "return true if number is even" in {
-
-    eval(" even(5) ") should be(ValBoolean(false))
-    eval(" even(2) ") should be(ValBoolean(true))
-  }
-
```

Comment:
I guess this change came from a different PR.

Reply:
When I merged my fork with the "new changes to upstream/master, this here test got added two times in my code base.

Reply:
Hmm. :thinking: I'm afraid that the test will be deleted when the PR is merged.

Reply:
I have added this here test again.. I don't know why it got lost in the first place. But now it is back.